config/deployments/env_specific.toml

```diff
@@ -151,6 +151,7 @@ redis_expiry = 900          # Redis expiry time in milliseconds
 [proxy]
 http_url = "http://proxy_http_url"    # Outgoing proxy http URL to proxy the HTTP traffic
 https_url = "https://proxy_https_url" # Outgoing proxy https URL to proxy the HTTPS traffic
+idle_pool_connection_timeout = 36000  # Idle pool connection timeout in milliseconds
```

Comment:
Let's remove this, this was one of our developers trying to debug some errors. From what I can find, this value will be used regardless of whether the user sets up a proxy or not, and using such a large value may not be preferable if the user does not set up a proxy.